src/components/Projects/WBS/SingleTask/SingleTask.js

```diff
@@ -0,0 +1,204 @@
+              </Button>
+            </NavItem>
+            <div id="single_task_name">{task.taskName}</div>
+          </ol>
+        </nav>
+
+        <table className="table table-bordered tasks-table">
+          <thead>
+            <tr>
+              <th scope="col" data-tip="task-num" colSpan="1">
+                #
+              </th>
+              <th scope="col" data-tip="Task Name" className="task-name">
+                Task Name
+              </th>
+              <th scope="col" data-tip="Priority">
+                <i className="fa fa-star" aria-hidden="true"></i>
+              </th>
+              <th className="desktop-view" scope="col" data-tip="Resources">
+                <i className="fa fa-users" aria-hidden="true"></i>
+              </th>
+              <th scope="col" data-tip="Assigned">
+                <i className="fa fa-user-circle-o" aria-hidden="true"></i>
+              </th>
+              <th className="desktop-view" scope="col" data-tip="Status">
+                <i className="fa fa-tasks" aria-hidden="true"></i>
+              </th>
+              <th className="desktop-view" scope="col" data-tip="Hours-Best">
+                <i className="fa fa-hourglass-start" aria-hidden="true"></i>
+              </th>
+              <th className="desktop-view" scope="col" data-tip="Hours-Worst">
+                <i className="fa fa-hourglass" aria-hidden="true"></i>
+              </th>
+              <th className="desktop-view" scope="col" data-tip="Hours-Most">
+                <i className="fa fa-hourglass-half" aria-hidden="true"></i>
+              </th>
+              <th className="desktop-view" scope="col" data-tip="Estimated Hours">
+                <i className="fa fa-clock-o" aria-hidden="true"></i>
+              </th>
+              <th className="desktop-view" scope="col" data-tip="Hours-Logged">
+              <i className="fa fa-hourglass-end" aria-hidden="true"></i>
+              </th>
+              <th className="desktop-view" scope="col" data-tip="Start Date">
+                <i className="fa fa-calendar-check-o" aria-hidden="true"></i> Start
+              </th>
+              <th className="desktop-view" scope="col" data-tip="Due Date">
+                <i className="fa fa-calendar-times-o" aria-hidden="true"></i> End
+              </th>
+              <th className="desktop-view" scope="col" data-tip="Links">
+                <i className="fa fa-link" aria-hidden="true"></i>
+              </th>
+              <th className="desktop-view" scope="col" data-tip="Details">
+                <i className="fa fa-question" aria-hidden="true"></i>
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <th scope="row">{task.num}</th>
+              <td>{task.taskName}</td>
+              <td>{task.priority}</td>
+              <td className="desktop-view">
+                {task.resources
```

Comment:
```suggestion
                {task?.resources &&
```
Also make this optionally chained, since you are testing for the object property.

You can use the `&&` and not need to return null in this case.

Reply:
Thanks for that review! I'll do that!